Doc/howto/pyporting.rst

```diff
@@ -20,409 +23,19 @@ How to port Python 2 Code to Python 3
 
    The archived python-porting_ mailing list may contain some useful guidance.
 
+   Since Python 3.13 the original official pyporting guide was discontinued.
+   You can still find the old guide in the
+   `archive <https://docs.python.org/3.12/howto/pyporting.html>`_.
```

Comment:
```suggestion
   Since Python 3.13 the original porting guide was discontinued.
   You can find the old guide in the
   `archive <https://docs.python.org/3.12/howto/pyporting.html>`_.
```